.tekton/prometheus-pull-request.yaml

```diff
@@ -33,9 +30,7 @@ spec:
   - name: build-platforms
     value:
     - linux/x86_64
-    - linux/arm64
     - linux/ppc64le
-    - linux/s390x
```

Comment:
Should we just keep this list instead of changing the default further down?

Reply:
We could do that, but let's also add arm64 and s390x